### PR TITLE
Fixing grammar in Office Add-ins

### PR DESCRIPTION
**Pull request by dasani-madipalli**
Made a few edits to the Office Add-in topic.

### CODE REVIEW

**Comment by dasani-madipalli** [comment]
@gregvanl here's a PR for an Office Add-in topic.

File: docs/other/office.md [hunk 1]
```diff
@@ -15,22 +15,22 @@ MetaDescription: See how to use the features of Visual Studio Code and integrate
...
+| Supported Office application | (Select Outlook) |
 | Create new add-in | Yes, I want a new add-in |
 | Add [TypeScript](https://www.typescriptlang.org/) | No |
 | Choose framework | Jquery |
 
-## Use Visual Studio Code to develop your Office Add-in!
+## Use Visual Studio Code to develop your Office Add-in
 
-Visual Studio Code is a great tool to help you develop your custom Office Add-ins regardless if they are for Outlook, Word, Excel, PowerPoint and run in the web clients, Windows clients, iOS clients or on macOS!
+Visual Studio Code is a great tool to help you develop your custom Office Add-ins for Outlook, Word, Excel, PowerPoint and runs in web clients, Windows clients, iOS clients or on macOS!
```
Comment (gregvanl) [inline docs/other/office.md:33]: I think this rewrite changes the meaning and implies that VS Code runs in web clients, Windows, iOS, and macOS, when the original sentence it talking about the Office Add-ins.

Comment (dasani-madipalli) [reply docs/other/office.md:33]: Ah got it - I actually originally interpreted it that way but this makes more sense. Perhaps, it's worth splitting this into two separate sentences.

Visual Studio Code is a great tool to help you develop your custom Office Add-ins for Outlook, Word, Excel, PowerPoint. It also enables you to run your Office Add-ins in web clients, Windows clients, iOS clients or on macOS!

**Comment by gregvanl** [comment]
Thanks!

**Pull request merged**